src/sentry/runner/commands/backup.py

```diff
@@ -59,7 +60,90 @@ def pretty(self) -> str:
         return "\n".join(f.pretty() for f in self.findings)
 
 
-def validate(expect: JSONData, actual: JSONData) -> ComparatorFindings:
+class JSONScrubbingComparator(ABC):
+    """An abstract class that compares and then scrubs some set of fields that, by a more nuanced
+    definition that mere strict byte-for-byte equality, are expected to maintain some relation on
```

Comment:
```suggestion
    definition than mere strict byte-for-byte equality, are expected to maintain some relation on
```

Reply:
You are incredibly thorough at catching all of the the typos and editing mistakes I make!

Reply:
> the the typos

Now you're just messing with me! 😅

Reply:
Hahaha I was sure you wouldn't catch it! Pretty impressive how you can spot those so easily - my (and most peoples') eyes just glide right over `the the` and other similar brain-fooling illusions.

Reply:
I did glide over it the first time! Otherwise I would've mentioned it in my first comment. My tendency is to pore over things, which slows me down but I guess means I catch more typos than average. Tradeoffs! 🙄 🤷‍♂️